apps/fluent-tester/src/FluentTester/TestComponents/Text/CustomizeUsage.tsx

```diff
@@ -13,11 +13,31 @@ export const CustomizeUsage: React.FunctionComponent<{}> = () => {
   const IndigoHeroBold = Text.customize({ tokens: { variant: 'heroStandard', fontWeight: '700', color: '#4b0082' } });
   const PurpleHeroLargeBold = Text.customize({ tokens: { variant: 'heroLargeStandard', fontWeight: '700', color: '#8402c4' } });
 
-  const ArialBlack = Text.customize({ tokens: { variant: 'heroLargeStandard', fontFamily: 'Arial Black' } });
+  // Android has a limited fontFamily support and unknown fonts fallback to sans-serif (Roboto).
+  // Custom fonts (ttf, otf) are supported but need to be linked using 'react-native link'
+  // The supported font families can be found as <aliases> at https://android.googlesource.com/platform/frameworks/base/+/master/data/fonts/fonts.xml
+  const CustomFontNames = {
+    ...Platform.select({
+      android: {
+        ArialBlack: 'arial',
+        CourierNew: 'courier new',
+        Georgia: 'georgia',
+        TimesNewRoman: 'times new roman',
+      },
+      default: {
+        ArialBlack: 'Arial Black',
+        CourierNew: 'Courier New',
+        Georgia: 'Georgia',
+        TimesNewRoman: 'Times New Roman',
+      },
+    }),
+  };
+
+  const ArialBlack = Text.customize({ tokens: { variant: 'heroLargeStandard', fontFamily: CustomFontNames.ArialBlack } });
   const BrushScriptMT = Text.customize({ tokens: { variant: 'heroStandard', fontFamily: 'Brush Script MT' } });
```

Comment:
If BrushScriptMT and Wingdings aren't available on Android, we should probably remove or replace them with fonts that are available on all platforms so feel free to change them!

Reply:
Along the same lines, I'd probably change the ArialBlack text to just Arial since android doesn't support ArialBlack.

Reply:
Added usage of supported fonts ComingSoon and Arial for Android.